dev-app-post/main/default/classes/LightningSelfRegisterControllerTest.cls

```diff
@@ -10,10 +10,10 @@ public with sharing class LightningSelfRegisterControllerTest {
   System.assert(LightningSelfRegisterController.isValidPassword(null, 'fakePwd') == false);
  }
 
- @IsTest
```

Comment:
@RanadheerRG When there is no test method, why do we need the @isTest annotation?

Reply:
![image](https://user-images.githubusercontent.com/120428823/232861836-acd8f7c0-4579-4800-ad0b-934c95e0e156.png)
I guess this is a github thing, all good.